naughty/fedora-42/8325-testing-branch-selinux-policy

```diff
@@ -0,0 +1,5 @@
+Traceback (most recent call last):
```

Comment:
Is that this particular message?

> type=AVC msg=audit(1758938947.568:397): avc:  denied  { read } for  pid=2423 comm="sshd-session" name="authorized_keys" dev="vda4" ino=110469 scontext=system_u:system_r:sshd_session_t:s0-s0:c0.c1023 tcontext=unconfined_u:object_r:httpd_exec_t:s0 tclass=file permissive=1

Can you please add it to the pattern, so that we don't give that test too much carte blanche?